specification/securitydevops/resource-manager/Microsoft.SecurityDevOps/preview/2022-09-01-preview/securitydevops.json

```diff
@@ -0,0 +1,2692 @@
+      "get": {
+        "tags": [
+          "default"
+        ],
+        "summary": "Returns a list of monitored AzureDevOps Connectors.",
+        "operationId": "AzureDevOpsConnector_ListBySubscription",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AzureDevOpsConnectorListResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "AzureDevOpsConnector_ListBySubscription": {
+            "$ref": "./examples/AzureDevOpsConnectorListBySubscription.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.SecurityDevOps/operations": {
+      "get": {
+        "description": "Returns list of operations supported by the Microsoft.SecurityDevOps provider.",
+        "operationId": "Operations_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Operation details.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Operations_List": {
+            "$ref": "./examples/OperationsList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ActionableRemediation": {
+      "type": "object",
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/ActionableRemediationState"
+        },
+        "severityLevels": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
```

Comment:
Are severity levels intended to be an enumeration of fixed values?

Reply:
We get this from SARIF https://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html but customers may have different types so have kept this as a list.